src/Template/Element/ui_language_button.ctp

```diff
@@ -4,8 +4,13 @@ use Cake\Core\Configure;
 if (!isset($displayOption)) {
     $displayOption = '';
 }
-$label = $this->Languages->getInterfaceLanguage();
+if (!isset($class)) {
+    $class = '';
+}
+if (!isset($label)) {
+    $label = $this->Languages->getInterfaceLanguage();
+}
 ?>
-<md-button <?= $displayOption ?> ng-click="showInterfaceLanguageSelection()">
+<md-button <?= $displayOption ?> class="<?= $class ?>" ng-click="showInterfaceLanguageSelection()">
```

Comment:
Pro tip: you can remove the above `isset` checks and then use the [null coalescing operator](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce):
```html
<md-button <?= $displayOption ?? '' ?> class="<?= $class ?? '' ?>" ...>
```